### PR TITLE
Mirror Firefox -> Firefox Android for css/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is created by a simple script that mirrors the compatibility data of all APIs from Firefox Desktop to Firefox Android when it is set to "null". This should help reduce inconsistencies in the browser data.

### CODE REVIEW

File: css/properties/position.json [hunk 1]
```diff
@@ -73,7 +73,8 @@
                 "notes": "Before Firefox 44, <code>position: fixed</code> didn't create a stacking context in most cases. Firefox and the specification have been modified to mimic Chrome and Safari's long-time behavior."
               },
               "firefox_android": {
-                "version_added": null
+                "version_added": "4",
```
Comment (Elchi3) [inline css/properties/position.json]: Changes to this file conflict with https://github.com/mdn/browser-compat-data/pull/3971 So maybe leave them out here?

**Comment by vinyldarkscratch** [comment]
Comments have been addressed, and changes have been pushed!

**Comment by Elchi3** [comment]
Thanks! There is still a change in positon.json for position: sticky that disagrees with https://github.com/mdn/browser-compat-data/pull/3971

**Comment by vinyldarkscratch** [comment]
Oops, never saved the reverts to the sticky positioning, fixed!

**Pull request merged**